warehouse/templates/packaging/detail.html

```diff
@@ -161,11 +161,11 @@ <h3 class="sidebar-section__title">GitHub Statistics</h3>
             <strong>Forks: </strong>
             <span class="github-repo-info__item" data-key="forks"></span>
           </p>
-          <p class="vertical-tabs__tab vertical-tabs__tab--with-icon vertical-tabs__tab--condensed">
+          <a class="vertical-tabs__tab vertical-tabs__tab--with-icon vertical-tabs__tab--condensed github-repo-info__item" data-key="html_url" data-attr="href">
```

Comment:
I think you'd want to add `rel="noopener" target="_blank"` to this anchor tag. I leave it to the Warehouse Devs to comment on whether the p -> a change is fine. :)

Reply:
I'm onboard with that. I noticed that some of the existing links don't open in a new tab so I kept it consistent.